lab_5_scrapper/scrapper.py

```diff
@@ -18,6 +71,16 @@ def __init__(self, path_to_config: pathlib.Path) -> None:
         Args:
             path_to_config (pathlib.Path): Path to configuration.
         """
+        self.path_to_config = path_to_config
+        self.config_DTO = self._extract_config_content()
+        self._validate_config_content()
+        self._num_articles = self.config_DTO.total_articles
+        self._seed_urls = self.get_seed_urls()
+        self._headers = self.config_DTO.headers
+        self._encoding = self.config_DTO.encoding
+        self._timeout = self.config_DTO.timeout
+        self._should_verify_certificate = self.config_DTO.should_verify_certificate
+        self._headless_mode = self.config_DTO.headless_mode
```

Comment:
Another questions, why store both DTO instance and all its properties as attributes? Isn't it redundant?